docs/development/extensions-contrib/azure.md

```diff
@@ -29,68 +29,15 @@ To use this Apache Druid extension, make sure to [include](../../development/ext
 
 [Microsoft Azure Storage](http://azure.microsoft.com/en-us/services/storage/) is another option for deep storage. This requires some additional Druid configuration.
 
-|Property|Possible Values|Description|Default|
+|Property|Description|Possible Values|Default|
 |--------|---------------|-----------|-------|
 |`druid.storage.type`|azure||Must be set.|
 |`druid.azure.account`||Azure Storage account name.|Must be set.|
 |`druid.azure.key`||Azure Storage account key.|Must be set.|
 |`druid.azure.container`||Azure Storage container name.|Must be set.|
-|`druid.azure.protocol`|http or https||https|
-|`druid.azure.maxTries`||Number of tries before cancel an Azure operation.|3|
+|`druid.azure.prefix`|prefix to use, i.e. what directory.| |""|
```

Comment:
Suggest:

"A prefix string that will be prepended to the blob names for the segments published to Azure deep storage"

Reply:
fixed